npm/private/npm_import.bzl

```diff
@@ -420,7 +420,12 @@ def _is_gnu_tar(rctx):
     if repo_utils.is_linux(rctx):
         return True
 
-    result = rctx.execute(["tar", "--version"])
+    tar_args = ["tar", "--version"]
```

Comment:
TODO to use hermetic tar toolchain?

Reply:
In theory we could, but tar happens to ship by default on macos, every major linux distro and even on Windows now. But I'll drop a TODO there.

Reply:
Benefit of using a hermetic tar would be that we don't have to check the version and can remote the "if gnu-tar" branch that adds an additional flag.

Reply:
Addressed in https://github.com/aspect-build/rules_js/pull/1460